### PR TITLE
fix: Changes to fix ee jest test

### PR DESCRIPTION
**Pull request by Rishabh-Rathod**
## Description

Changes to fix the jest test failure on EE. 

## Type of change

- Chore

### Dev activity
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

### QA activity:
- [ ] Test plan has been approved by relevant developers
- [ ] Test plan has been peer reviewed by QA
- [ ] Cypress test cases have been added and approved by either SDET or manual QA
- [ ] Organized project review call with relevant stakeholders after Round 1/2 of QA
- [ ] Added Test Plan Approved label after reveiwing all Cypress test

### CODE REVIEW

**Review by berzerkeer — APPROVED** [review]

File: app/client/src/workers/Evaluation/fns/utils/TriggerEmitter.ts [hunk 1]
```diff
@@ -146,4 +147,13 @@ TriggerEmitter.on(
...
 );
 
+export const fnInvokeLogHandler = priorityBatchedActionHandler((data) => {
+  WorkerMessenger.ping({
+    method: MAIN_THREAD_ACTION.LOG_JS_FUNCTION_EXECUTION,
+    data,
+  });
+});
+
+TriggerEmitter.on(BatchKey.process_batched_fn_invoke_log, fnInvokeLogHandler);
```
Comment (ravikp7) [inline app/client/src/workers/Evaluation/fns/utils/TriggerEmitter.ts:157]: @Rishabh-Rathod I'm assuming this is a newly added event `process_batched_fn_invoke_log`. I want to understand where this event is being triggered from.

Comment (Rishabh-Rathod) [reply app/client/src/workers/Evaluation/fns/utils/TriggerEmitter.ts:157]: This is getting triggered in the EE repo code in `postJSFunctionExecution`  https://github.com/appsmithorg/appsmith-ee/blob/release/app/client/src/ee/workers/Evaluation/JSObject/postJSFunctionExecution.ts

Once this PR is merged i would move that code to TriggerEmitter to resolve the cyclic import issue in jest test.

**Pull request merged**